src/commands/node.mjs

```diff
@@ -2314,8 +2352,829 @@ export class NodeCommand extends BaseCommand {
+           * @property {string} freezeAdminPrivateKey
+           * @property {string} keysDir
+           * @property {Object} nodeClient
+           * @property {Object} podNames
+           * @property {Map<String, NetworkNodeServices>} serviceMap
+           * @property {PrivateKey} treasuryKey
+           * @property {string} stagingDir
+           * @property {string} stagingKeysDir
+           * -- methods --
+           * @property {getUnusedConfigs} getUnusedConfigs
+           */
+          /**
+           * @callback getUnusedConfigs
+           * @returns {string[]}
+           */
+
+          // create a config object for subsequent steps
+          const config = /** @type {NodeUpdateConfigClass} **/ this.getConfig(NodeCommand.UPDATE_CONFIGS_NAME, NodeCommand.UPDATE_FLAGS_LIST,
+            [
+              'adminKey',
+              'allNodeIds',
+              'existingNodeIds',
+              'freezeAdminPrivateKey',
+              'keysDir',
+              'nodeClient',
+              'podNames',
+              'serviceMap',
+              'stagingDir',
+              'stagingKeysDir',
+              'treasuryKey'
+            ])
+
+          config.curDate = new Date()
+          config.existingNodeIds = []
+
+          await self.initializeSetup(config, self.k8)
+
+          // set config in the context for later tasks to use
+          ctx.config = config
+
+          ctx.config.chartPath = await self.prepareChartPath(ctx.config.chartDirectory,
+            constants.FULLSTACK_TESTING_CHART, constants.FULLSTACK_DEPLOYMENT_CHART)
+
+          // initialize Node Client with existing network nodes prior to adding the new node which isn't functioning, yet
+          ctx.config.nodeClient = await this.accountManager.loadNodeClient(ctx.config.namespace)
+
+          const accountKeys = await this.accountManager.getAccountKeysFromSecret(FREEZE_ADMIN_ACCOUNT, config.namespace)
+          config.freezeAdminPrivateKey = accountKeys.privateKey
+
+          const treasuryAccount = await this.accountManager.getTreasuryAccountKeys(config.namespace)
+          const treasuryAccountPrivateKey = treasuryAccount.privateKey
+          config.treasuryKey = PrivateKey.fromStringED25519(treasuryAccountPrivateKey)
+
+          self.logger.debug('Initialized config', { config })
+        }
+      },
+      {
+        title: 'Identify existing network nodes',
+        task: async (ctx, task) => {
+          const config = /** @type {NodeAddConfigClass} **/ ctx.config
+          config.serviceMap = await self.accountManager.getNodeServiceMap(
+            config.namespace)
+          for (/** @type {NetworkNodeServices} **/ const networkNodeServices of config.serviceMap.values()) {
+            config.existingNodeIds.push(networkNodeServices.nodeName)
+          }
+
+          return self.taskCheckNetworkNodePods(ctx, task, config.existingNodeIds)
+        }
+      },
+      {
+        title: 'Determine new node account number',
```

Comment:
I don't see a reason to change the node account number to the next in sequence.  I don't think this would ever happen in the real world, and the old number would now create a gap.

Reply:
This is needed to get a bew account id used in NodeUpdateTransaction,
for a three node network, 
changing 0.0.3 to 0.0.6, I think I will change to use a flag parameter to get a new account number.

Meanwhile, what could be problem if account number has a gap ? ( not a node id gap)

Reply:
I talked with Nathan, this is valid.  but, we should make a flag to decide whether or not to change the account number.

Reply:
Updated